apps/dolly-frontend/src/main/js/src/components/bestillingsveileder/stegVelger/steg/steg1/paneler/Personinformasjon.tsx

```diff
@@ -126,33 +111,46 @@ PersoninformasjonPanel.initialValues = ({ set, setMulti, del, has, opts }) => {
 		},
 		doedsdato: {
 			label: 'Dødsdato',
-			checked: has('tpsf.doedsdato'),
-			add: () => set('tpsf.doedsdato', null),
-			remove: () => del(['tpsf.doedsdato', 'tpsf.alder', 'tpsf.foedtEtter', 'tpsf.foedtFoer']),
+			checked: has('pdldata.person.doedsfall'),
+			add: () =>
+				set('pdldata.person.doedsfall', [
+					{
+						doedsdato: new Date(),
+						kilde: 'Dolly',
+						master: 'PDL',
+						gjeldende: true,
+					},
+				]),
+			remove: () => del(['pdldata.person.doedsfall']),
 		},
 		statsborgerskap: {
 			label: 'Statsborgerskap',
-			checked: has('tpsf.statsborgerskap'),
+			checked: has('pdldata.person.statsborgerskap'),
 			add() {
-				setMulti(
-					['tpsf.statsborgerskap', ''],
-					['tpsf.statsborgerskapRegdato', null],
-					['tpsf.statsborgerskapTildato', null]
-				)
+				_has(personFoerLeggTil, 'pdlforvalter[0].person.statsborgerskap')
+					? set('pdldata.person.statsborgerskap', fjernIdFoerLeggTil('statsborgerskap'))
+					: set('pdldata.person.statsborgerskap', [
+							{
+								landkode: null,
+								gyldigFraOgMed: new Date(),
+								gyldigTilOgMed: null,
+								bekreftelsesdato: null,
+								kilde: 'Dolly',
+								master: 'PDL',
+								gjeldende: true,
+							},
+					  ])
 			},
 			remove() {
-				del(
-					['tpsf.statsborgerskap', 'tpsf.statsborgerskapRegdato'],
-					['tpsf.statsborgerskap', 'tpsf.statsborgerskapTildato']
-				)
+				del(['pdldata.person.statsborgerskap'])
 			},
 		},
 		innvandretFraLand: {
 			label: 'Innvandret fra',
 			checked: has('pdldata.person.innflytting'),
 			add() {
 				_has(personFoerLeggTil, 'pdlforvalter[0].person.innflytting')
-					? set('pdldata.person.innflytting', flyttingFoerLeggTil('innflytting'))
+					? set('pdldata.person.innflytting', fjernIdFoerLeggTil('innflytting'))
```

Comment:
Hva betyr fjernIdFoerLeggTil?

Reply:
Du skal kunne ha flere innflyttinger ...

Reply:
fjernIdFoerLeggTil bare fjerner ID feltet fra det som hentes, siden det ikke skal være med i formet frontend 👍  Det skal være satt opp mulighet for flere inflyttinger ja 😉